v_4_9_1/master_template.go

```diff
@@ -362,18 +362,6 @@ systemd:
       [Install]
       WantedBy=multi-user.target
 
-  - name: debug-tools.service
```

Comment:
This service was complicating the process and it's value is questionable for me. If we have SSH access, we can SFTP these binaries in instead.

Reply:
We already changed this in v_5_X_X - instead of removing the whole unit, please just set enabled to false.

If you have network issues, it is very cumbersome IMO to use SFTP. A disabled unit that can just be enabled when debugging is easier to deal with.